test/jdk/com/sun/nio/sctp/SctpChannel/CloseDescriptors.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8238274
+ * @summary Potential leak file descriptor for SCTP
+ * @requires (os.family == "linux")
+ * @run main CloseDescriptors
```

Comment:
The test SctpMultiChannel/CloseDescriptors seems to work - at least it doesn't fail all the time - but it's using `@run main/othervm`. I believe you should make this test run in /othervm mode for more stability. I will test again with your later change and /othervm mode.

Reply:
I pushed a fix to run the test in othervm mode. Please test again.

Reply:
Thank you for your help. I applied your suggested fix and it works fine. But it takes about 4 minutes to be finished. So I added a timeout parameter to 250.